scan_test.go:testScanSuite.TestScan failed

scan_test.go:testScanSuite.TestScan

```
[2020-08-07T03:26:57.316Z] ----------------------------------------------------------------------
[2020-08-07T03:26:57.316Z] FAIL: scan_test.go:65: testScanSuite.TestScan
[2020-08-07T03:26:57.316Z]
[2020-08-07T03:26:57.316Z] scan_test.go:110:
[2020-08-07T03:26:57.316Z]     check(c, scan, rowNum, false)
[2020-08-07T03:26:57.316Z] scan_test.go:69:
[2020-08-07T03:26:57.316Z]     c.Assert([]byte(k), BytesEquals, encodeKey(s.prefix, s08d("key", i)))
[2020-08-07T03:26:57.316Z] ... bytes_one []uint8 = []byte{0x73, 0x65, 0x65, 0x6b, 0x5f, 0x31, 0x35, 0x39, 0xff, 0x36, 0x37, 0x37, 0x30, 0x36, 0x37, 0x34, 0x5f, 0xff, 0x6b, 0x65, 0x79, 0x30, 0x30, 0x30, 0x30, 0x30, 0xff, 0x34, 0x32, 0x35, 0x0, 0x0, 0x0, 0x0, 0x0, 0xfa}
[2020-08-07T03:26:57.316Z] ... bytes_two []uint8 = []byte{0x73, 0x65, 0x65, 0x6b, 0x5f, 0x31, 0x35, 0x39, 0xff, 0x36, 0x37, 0x37, 0x30, 0x36, 0x37, 0x34, 0x5f, 0xff, 0x6b, 0x65, 0x79, 0x30, 0x30, 0x30, 0x30, 0x30, 0xff, 0x33, 0x37, 0x39, 0x0, 0x0, 0x0, 0x0, 0x0, 0xfa}
[2020-08-07T03:26:57.316Z]
```

 Latest failed builds: 
https://internal.pingcap.net/idc-jenkins/job/tidb_ghpr_unit_test/45604/display/redirect

```
[2020-08-20T08:24:12.121Z] [2020/08/20 16:22:39.382 +08:00] [ERROR] [scan_test.go:79] ["bytes equal check fail"] [i=437] [rowNum=768] ["obtained key"=7480000000000000015f7280000000000001b8] ["obtained val"=80343430] [expected=7480000000000000015f7280000000000001b5] [keyOnly=false]

[2020-08-20T08:24:12.121Z] ... bytes_one []uint8 = []byte{0x74, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x5f, 0x72, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xb8}
[2020-08-20T08:24:12.121Z] ... bytes_two []uint8 = []byte{0x74, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x5f, 0x72, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xb5}
```

```
[2020-08-20T08:24:12.121Z] [2020/08/20 16:22:39.382 +08:00] [ERROR] [scan_test.go:79] ["bytes equal check fail"] [i=437] [rowNum=768] ["obtained key"=7480000000000000015f7280000000000001b8] ["obtained val"=80343430] [expected=7480000000000000015f7280000000000001b5] [keyOnly=false]

[2020-08-20T08:47:03.179Z] ... bytes_one []uint8 = []byte{0x74, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x5f, 0x72, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xb9}
[2020-08-20T08:47:03.179Z] ... bytes_two []uint8 = []byte{0x74, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x5f, 0x72, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xb4}
```

The new test failure cases, the row value is not expected value, need more investigations.

Reproduce on the local environment(not very frequent, possible 1/100), the scan next call returned value is incorrect.

```
case 1 from 380
[ERROR] [scan_test.go:79] ["bytes equal check fail"] [i=380] [rowNum=768] ["obtained key"=7480000000000000015f728000000000000185] ["obtained val"=80333839] [expected=7480000000000000015f72800000000000017c] [keyOnly=false]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f728000000000000185] [j=380]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f728000000000000190] [j=381]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f72800000000000019c] [j=382]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000001a6] [j=383]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000001b5] [j=384]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000001b6] [j=385]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000001b7] [j=386]

case 2 from 713
[ERROR] [scan_test.go:79] ["bytes equal check fail"] [i=713] [rowNum=768] ["obtained key"=7480000000000000015f7280000000000002d2] ["obtained val"=80373232] [expected=7480000000000000015f7280000000000002c9] [keyOnly=false]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002d2] [j=713]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002dd] [j=714]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002e6] [j=715]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002ed] [j=716]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002f5] [j=717]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002fd] [j=718]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002fe] [j=719]
[INFO] [scan_test.go:88] [key] [key=7480000000000000015f7280000000000002ff] [j=720]
```

--- a/go.mod
+++ b/go.mod
@@ -20,11 +20,10 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de4
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
 	github.com/juju/errors v0.0.0-20181118221551-089d3ea4e4d5
-	github.com/juju/testing v0.0.0-20200706033705-4c23f9c453cd // indirect
 	github.com/klauspost/cpuid v1.2.1
 	github.com/ngaut/pools v0.0.0-20180318154953-b7bc8c42aac7
 	github.com/ngaut/sync2 v0.0.0-20141008032647-7a24ed77b2ef
-	github.com/ngaut/unistore v0.0.0-20200820080223-c734bcc4ea53
+	github.com/ngaut/unistore v0.0.0-20200826054345-a70ab8a1f698
 	github.com/opentracing/basictracer-go v1.0.0
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2

